rider-core/src/main/java/com/github/database/rider/core/configuration/DataSetConfig.java

```diff
@@ -221,21 +230,26 @@ public void setTransactional(boolean transactional) {
     public String toString() {
         StringBuilder sb = new StringBuilder();
         for (String dataset : datasets) {
-            sb.append(dataset).append(",");
+            sb.append(dataset).append(", ");
+        }
+        if(provider != null && !provider.isInterface()) {
```

Comment:
Duplicated condition in several places

Reply:
You mean that we should remove the check here? https://github.com/rmpestano/database-rider/blob/61961978ea06952c847ece74042757d1782589e7/rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java#L149

Reply:
Just created a dedicated method. [see here](https://github.com/database-rider/database-rider/blob/f964f8d36bc9b5e921101ee62f9df90bc663198c/rider-core/src/main/java/com/github/database/rider/core/configuration/DataSetConfig.java#L248)